tests/test_inventory/group_vars/all.yml

```diff
@@ -0,0 +1,79 @@
+# Directory where the binaries will be installed
```

Comment:
You're right. I successfully tested pushing the generated test inventory into the inventory directory. I'm going to remove this useless file from the PR